apisix/plugins/jwt-auth.lua

```diff
@@ -252,15 +253,12 @@ end
 local function get_secret(conf, consumer_name)
     local secret = conf.secret
     if conf.vault then
```

Comment:
Just delete the vault-related configuration directly, because we have already retrieved and replaced the configuration of all auth plugins (type = "auth") in the consumer.
https://github.com/apache/apisix/blob/master/apisix/consumer.lua#L107